### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by brunobat**

### CODE REVIEW

File: quarkus-opentelemetry-exporter-azure/deployment/pom.xml [hunk 1]
```diff
@@ -22,6 +22,18 @@
...
+            <artifactId>quarkus-jackson-deployment</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jaxb-deployment</artifactId>
+        </dependency>
+<!--        <dependency>-->
+<!--            <groupId>io.quarkiverse.azureservices</groupId>-->
+<!--            <artifactId>quarkus-azure-http-client-vertx-deployment</artifactId>-->
+<!--        </dependency>-->
```
Comment (jeanbisutti) [inline quarkus-opentelemetry-exporter-azure/deployment/pom.xml:36]:
```suggestion
```

File: pom.xml [hunk 2]
```diff
@@ -38,20 +38,21 @@
 
         <!-- On this block, library versions must be kept in sync with the ones in the quarkus-bom: -->
         <!-- Start-->
-        <quarkus.version>3.5.3</quarkus.version>
+        <quarkus.version>3.8.3</quarkus.version>
         <netty-bom.version>4.1.107.Final</netty-bom.version>
-        <opentelemetry.version>1.31.0</opentelemetry.version>
-        <opentelemetry-alpha.version>1.31.0-alpha</opentelemetry-alpha.version>
-        <smallrye-config.version>3.5.4</smallrye-config.version>
+        <opentelemetry.version>1.32.0</opentelemetry.version>
```
Comment (jeanbisutti) [inline pom.xml:43]: It would be great to use the last OTel version in the Quarkus OTel extension and in the Quarkus OTel exporters ;)

**Review by jeanbisutti — APPROVED** [review]
Great! 👍

File: pom.xml (hunk 2 above)
Comment (brunobat) [reply pom.xml:43]: Not going to happen soon because of semantic conventions.

**Pull request merged**